fido/fido.py

```diff
@@ -350,11 +364,12 @@ def get_extension(self, format):
         return format.find('extension').text
 
     def print_matches(self, fullname, matches, delta_t, matchtype=''):
-        """The default match handler.  Prints out information for each match in the list.
-           @param fullname is name of the file being matched
-           @param matches is a list of (format, signature)
-           @param delta_t is the time taken for the match.
-           @param matchtype is the type of match (signature, containersignature, extension, fail)
+        """
+        The default match handler. Prints out information for each match in the list.
```

Comment:
I know! I actually added pep257 in travis but commented it out for now. But you're right, I will work on it, I was a bit worried about the timeline for the release but I will make a little effort and fix all the warnings thrown by the 257 linter. Thanks.